src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -195,9 +204,9 @@ public function install( $args, $assoc_args ) {
 					$errors++;
 				} elseif ( is_wp_error( $result ) ) {
 					$key = $result->get_error_code();
-					if ( in_array( $key, array( 'plugins_api_failed', 'themes_api_failed' ) )
-						&& ! empty( $result->error_data[ $key ] ) && in_array( $result->error_data[ $key ], array( 'N;', 'b:0;' ) ) ) {
-						\WP_CLI::warning( "Couldn't find '$slug' in the WordPress.org {$this->item_type} directory." );
+					if ( in_array( $key, [ 'plugins_api_failed', 'themes_api_failed' ], true )
+						&& ! empty( $result->error_data[ $key ] ) && in_array( $result->error_data[ $key ], [ 'N;', 'b:0;' ], true ) ) {
```

Comment:
I am unsure about this check over here, `in_array( $result->error_data[ $key ], [ 'N;', 'b:0;' ], true )`, I tried to install non existent plugin, but was not able to get this result.

Reply:
Here's the commit in question. There's an associated test that should still pass then...

Reply:
Forgot the actual link: https://github.com/wp-cli/extension-command/commit/cf8631cb6d800ae4dfd32221735a93f5667406c0

Reply:
Ok, only the test was changed in https://github.com/wp-cli/extension-command/pull/85 to adopt a new version of API.

Reply:
Leave it in with the strict check for now, It didn't seem to break something up until now, and I don't think the type can be off in this case.